backtrace-library/src/main/java/backtraceio/library/BacktraceDatabase.java

```diff
@@ -92,6 +117,30 @@ private String getDatabasePath() {
         return databaseSettings.getDatabasePath();
     }
 
+    /**
+     * Setup native crash handler
+     * @param client  Backtrace client
+     * @param credentials Backtrace credentials
+     */
+    public void setupNativeIntegration(BacktraceBase client, BacktraceCredentials credentials) {
+        // avoid initialization when database doesn't exist
+        if(getSettings() == null) {
+            return;
+        }
+        // path to crashpad native handler
+        String handlerPath = _applicationContext.getApplicationInfo().nativeLibraryDir + "/libcrashpad_handler.so";
+
+        BacktraceAttributes crashpadAttributes = new BacktraceAttributes(_applicationContext, null, client.attributes);
+        List<String> result = new ArrayList(crashpadAttributes.attributes.keySet());
+        List<String> values = new ArrayList(crashpadAttributes.attributes.values());
+        Initialize(
+                credentials.getMinidumpSubmissionUrl().toString(),
+                getSettings().getDatabasePath() + "/crashpad",
```

Comment:
I would be in favor of extracting this string "/crashpad" to the separate final variable in this class

Reply:
done